.circleci/config.yml

```diff
@@ -0,0 +1,219 @@
+version: 2
+jobs:
+    build:
+        working_directory: ~/fxa
+        docker:
+        - image: circleci/node:8
+          env:
+          - DISPLAY=:99
+
+        steps:
+        - checkout
+        - run:
+              command: |
+                  sudo apt-get install -y python-setuptools python-dev build-essential xvfb
+                  sudo easy_install pip
+                  sudo pip install mozdownload mozinstall
+                  mozdownload --version 58.0 --destination firefox.tar.bz2
+                  mozinstall firefox.tar.bz2
+                  sudo npm install -g npm@6
+
+                  cp server/config/local.json-dist server/config/local.json
+                  npm i --production
+
+                  CONFIG_FILES=server/config/local.json,server/config/production.json,tests/ci/config_circleci.json node_modules/.bin/grunt build
+                  npm install intern xmlhttprequest firefox-profile request sync-exec convict yargs leadfoot otplib retry-cli
+                  ln -s node_modules/.bin/retry retry
+
+        - persist_to_workspace:
+              root: ~/
+              paths: ./*
+
+    test1:
+        environment:
+            CIRCLE_NODE_INDEX: 0
+            CIRCLE_NODE_TOTAL: 4
+
+        working_directory: ~/fxa
+        docker:
+            - image: circleci/node:8-stretch-browsers
```

Comment:
Curious why this is different from the one in the build step?

Reply:
This images comes with dependencies require for browser testing. See https://circleci.com/docs/2.0/browser-testing/#overview